src/app/components/profile/profile.component.ts

```diff
@@ -17,48 +16,46 @@ import { InputComponent } from '../utility/input/input.component';
   styleUrls: ['./profile.component.scss']
 })
 export class ProfileComponent implements OnInit {
-
   /**
    * User object
    */
   user: any;
+  first_name: any;
+  last_name: any;
+  affiliation: any;
+  google_scholar_url: any;
+  github_url: any;
+  linkedin_url: any;
 
   /**
    * Profile completion score
    */
   pcomp: any;
-
```

Comment:
I still can see these changes, please revert this and similar changes.

Reply:
which changes?

Reply:
You have these blank lines here

Reply:
a blank line between each variable?